python/ray/rllib/test/test_multi_agent_env.py

```diff
@@ -64,7 +70,7 @@ def __init__(self, num, increment_obs=False):
         self.last_info = {}
         self.i = 0
         self.num = num
-        self.observation_space = gym.spaces.Discrete(2)
+        self.observation_space = gym.spaces.Discrete(10)
```

Comment:
is there a test for the multi-agent preprocessing?

Reply:
Yeah test_nested_spaces.